lib/scanner/blocks.go

```diff
@@ -87,14 +94,14 @@ func Blocks(ctx context.Context, r io.Reader, blocksize int, sizehint int64, cou
 			Size:     int32(n),
 			Offset:   offset,
 			Hash:     thisHash,
-			WeakHash: whf.Sum32(),
+			WeakHash: hashFun.Sum32(),
 		}
 
 		blocks = append(blocks, b)
 		offset += n
 
 		hf.Reset()
-		whf.Reset()
+		hashFun.Reset()
```

Comment:
hashFun and hf mean the same thing, yet are different things.
I'd call them strongHash and weakHash

Reply:
yes, I had the same idea